hare/integration_test.go

```diff
@@ -82,8 +82,8 @@ type hareIntegration20Nodes struct {
 }
 
 func Test_20Nodes_HareIntegrationSuite(t *testing.T) {
+	t.Skip()
```

Comment:
revert this

Reply:
Skip for now (not used anyway for testing)